examples/protocol_specification_ex/sample.yaml

```diff
@@ -15,6 +15,7 @@ speech_acts:
     proposal: pt:optional[pt:dict[ct:IOTApp7, pt:bytes]]
     rounds: pt:set[pt:int]
     items: pt:list[ct:Unit]
+    conditions: pt:optional[pt:union[pt:str, pt:dict[pt:str,pt:int], pt:set[ct:DataModel], pt:dict[pt:bytes, pt:float]]]
```

Comment:
Is this possible to make it in the form of a dictionary in the future? It is kind of difficult
to read :)

Reply:
That's just to test the functionality. Any developer who is creating a protocol specification is responsible for its complexity. 

The sample specification is just to show (and test) various possibilities.